Jenkinsfile

```diff
@@ -477,7 +479,70 @@ EOL
                 }
             }
         }
-	}
+        stage('Package helm charts'){
+            steps{
+                dir ("${WORKSPACE}/deploy/helms") {
+                    script {
+                        chartNames = sh(returnStdout: true, script: 'ls').split()
+                        println (chartNames)
```

Comment:
Worth adding some description for the output if it's not abandoned debug which should be removed

Reply:
Added comment to this step.